Gems/Atom/RPI/Registry/atom_rpi.setreg

```diff
@@ -6,7 +6,8 @@
                     "CommonSrgsShaderAssetPath": "shaders/sceneandviewsrgs.azshader",
                     "ImageSystemDescriptor": {
                         "SystemStreamingImagePoolSize": 0,
-                        "SystemAttachmentImagePoolSize": 0 
+                        "SystemStreamingImagePoolMipBias": 1,
```

Comment:
Should this be 0 instead of 1?

Reply:
you are right. I forgot to change it back from testing